http/src/main/resources/init-resources/shutdown.sh

```diff
@@ -9,8 +9,13 @@ set -e -x
 # Templated values
 export RSTUDIO_DOCKER_IMAGE=$(rstudioDockerImage)
 export RSTUDIO_SERVER_NAME=$(rstudioServerName)
+export SHOULD_DELETE_JUPYTER_DIR=$(shouldDeleteJupyterDir)
 
 # If RStudio is installed, cleanly shut it down
 if [ ! -z "$RSTUDIO_DOCKER_IMAGE" ] ; then
     docker exec -u rstudio -i $RSTUDIO_SERVER_NAME rstudio-server stop
+fi
+
+if [ -d '/mnt/disks/work/.jupyter' ] && [ "SHOULD_DELETE_JUPYTER_DIR" = "true" ] ; then
```

Comment:
is this code tested somewhere? everywhere else in bash scripts (such as `gce-init.sh`) we use `== "true"`, might be worth updating

Reply:
it's indirectly validated by having that PD test using a different image...cuz the goal is to make sure switching to a different image will still work

Reply:
sounds good - didn't know that we can use `=` and `==`!